GliaWidgets/Sources/ViewModel/Chat/ChatViewModel+GVA.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import GliaCoreSDK
+
+private extension String {
+    static let gvaOptionUrlTargetModal = "modal"
+    static let gvaOptionUrlTargetSelf = "self"
+}
+
+extension ChatViewModel {
+    func gvaOptionAction(for option: GvaOption) -> Cmd {
+        // If option contains `url`, then it's URL Button,
+        // otherwise it's Postback Button and option should be sent
+        // to the server as `SingleChoiceOption`
+        if let urlString = option.url,
+           let url = URL(string: urlString) {
+            return urlButtonAction(url: url, urlTarget: option.urlTarget)
+        } else {
```

Comment:
I would add another case for a broadcast event that logs an error. So if a broadcast event is present regardless of other values, then error should be logged but not presented

Reply:
Good point! 
I will add presenting an alert indicating that broadcast events are not supported as mentioned in project plan